R/internal.R

```diff
@@ -51,8 +51,27 @@ add_multiline_prefix <- function(prefix, as_is = FALSE) {
   )
 }
 
+
+
 enclose <- function(prefix, postfix = prefix) {
   a <- rstudioapi::getSourceEditorContext()
   for (s in a$selection)
     rstudioapi::insertText(location = s$range, text = sprintf("%s%s%s", prefix, s$text, postfix))
+  
+  return(a)
+}
+
+#' Function Wrap
```

Comment:
please fill in the documentation in the same fashion as other exported functions in the package including an example. you can see one [here](https://github.com/ThinkR-open/remedy/blob/master/contributing.md#youre-opening-a-pull-request) to work mimic. There must be an example for the package to be updated in CRAN.